### PR TITLE
TFW-3453 - added plotted lines to combo chart

### PR DESCRIPTION
**Pull request by troyblank**
This adds plotted lines on z index on combo.

Screen Shot 2019-05-03 at 10 32 57 AM

### CODE REVIEW

File: components/visualizations/colors.beta.scss [hunk 1]
```diff
@@ -27,6 +31,16 @@
 .color-vis-blue-8 { @include chart-color($color-vis-blue-8); }
 .color-vis-blue-9 { @include chart-color($color-vis-blue-9); }
 
+.stroke-vis-blue-1 { @include stroke-color($color-vis-blue-1); }
```
Comment (jasonhale) [inline components/visualizations/colors.beta.scss]: Would it help or hinder using Sass's `@for` loop here?
```@for $i from 1 through 9 {
  .stroke-vis-blue-#{$i} {
    @include stroke-color($color-vis-blue-#{$i});
  }
```

Just wondering if that'd be helpful, or even beneficial, now that there's 2 instances of sequential colors.

File: assets/js/mockData/comboBarLine.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ const comboBarLineData = {
         },
         z: {
             fields: [
-                'silentTime',
-                'duration'
+                'duration',
```
Comment (tVeratti) [inline assets/js/mockData/comboBarLine.js]: Is the order important?

File: components/visualizations/colors.beta.scss (hunk 1 above)
Comment (tVeratti) [reply components/visualizations/colors.beta.scss]: Nice idea, we could really get sass to do some of this work for us :)

File: components/visualizations/comboBarLine/comboBarLine.beta.scss [hunk 3]
```diff
@@ -34,3 +34,8 @@
 .combo-bar-line .tick text {
     transform: translate(3px, 0);
 }
+
+.combo-bar-line__line {
```
Comment (tVeratti) [inline components/visualizations/comboBarLine/comboBarLine.beta.scss]: combobarline line? :P

File: components/visualizations/colors.beta.scss (hunk 1 above)
Comment (troyblank) [reply components/visualizations/colors.beta.scss]: This is a great idea; but it appears making dynamic vars on run time is not supported by sass from my toying around. I could be wrong but it doesn't seem to be possible.

File: assets/js/mockData/comboBarLine.js (hunk 2 above)
Comment (troyblank) [reply assets/js/mockData/comboBarLine.js]: It is sorta, it matches up the colors in which in design silent time is less important (lighter blue)... we could make a color map that loosely connects the props though but order seemed more dynamic and iron glad to me. So yea it matters in which one gets the light or dark blue but it does not mater a lot. I talked to API peeps to make sure this order is like this.

File: components/visualizations/comboBarLine/comboBarLine.beta.scss (hunk 3 above)
Comment (troyblank) [reply components/visualizations/comboBarLine/comboBarLine.beta.scss]: david?

**Pull request merged**